spec/lib/miq_automation_engine/service_methods/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm_spec.rb

```diff
@@ -14,11 +14,11 @@ module MiqAeServiceVmVmwareSpec
         :task_id     => nil
       }
 
-      $_miq_worker_current_msg = FactoryGirl.build(:miq_queue, :task_id => '1234')
+      @_miq_worker_current_msg = FactoryGirl.build(:miq_queue, :task_id => '1234')
```

Comment:
This change is not doing what you think it does.  This is actually modifying the global $_miq_worker_current_msg for queue workers.  See [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_queue_worker_base/runner.rb#L105-L106) and/or just git grep for it.  Arguably, you could say that it's not needed at all as I'm not sure of it's purpose in this spec in the first place.